docs/source/rst/examples.rst

```diff
@@ -411,7 +411,7 @@ class. The order of the geometry objects in the list that is passed into the con
 order relates to specifying mesh sizes and material properties.
 
 Once the geometry has been merged, it is vital to clean the geometry to remove
-any artefacts that may impede the meshing algorithm. A mesh is created with a mesh
+any artifacts that may impede the meshing algorithm. A mesh is created with a mesh
```

Comment:
artefact is British English which is what I use :smile:

Reply:
changed back.